scripts/preprocess_VID_data.py

```diff
@@ -58,7 +58,6 @@ def process_split(root_dir, save_dir, split, subdir='', ):
 
       # Read image
       img_file = osp.join(data_dir, folder, filename + '.JPEG')
-      img = None
```

Comment:
We use `img = None` such that it is not loaded several times for each annotation.

We assumed the image files exist without further check. If they are not there, it is possible that the dataset has already been broken.  However, add this check will enable users to fully understand the reasons when it fails.